libs/sdk/skyux-eslint/README.md

```diff
@@ -0,0 +1,50 @@
+# skyux-eslint
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `nx test skyux-eslint` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Implement in eslint.config.js
+
+```
+// @ts-check
+const eslint = require("@eslint/js");
+const angular = require("angular-eslint");
+const skyux = require('skyux-eslint');
+const tseslint = require("typescript-eslint");
```

Comment:
Please use single quotes. Also, mention that this is in preview.

Reply:
Also, fix the capitalization of `tseslint`.

Reply:
Done.